src/07_slices.py

```diff
@@ -12,26 +12,26 @@
 a = [2, 4, 1, 7, 9, 6]
 
 # Output the second element: 4:
-print()
+print(a[1])
 
 # Output the second-to-last element: 9
-print()
+print(a[-2:-1])
```

Comment:
this will work, but if you leave the second argument blank  a[-2:] it will still slice to the end of the list.
the same is true for targeting the beginning of list, a[:5] slices a list from index 0 up to the 5th